.github/workflows/dotnet-desktop.yml

```diff
@@ -0,0 +1,42 @@
+name: .NET Core Desktop
+
+on:
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+
+  build:
+
+    strategy:
+      matrix:
+        configuration: [Debug, Release]
```

Comment:
Debug is not needed

Reply:
And also, you don't use this configuration anywhere 
looks like it can be deleted